docs/user-guide/environment-variables.md

```diff
@@ -9,6 +9,8 @@ toc:
       active: true
     - title: Build Specific
       url: "#build-specific"
+    - title: Plugins
```

Comment:
Can we call this `Services` ? Plugins gives Jenkins connotations.

Reply:
I think we call it plugins everywhere already
https://docs.screwdriver.cd/about/contributing/where-to-contribute, http://screwdriver.cd
Did we want to change everything?

Reply:
Better not, I guess I can reason. `Screwdriver` interfaces with various external Services via `Plugins`.